### ISSUE TITLE
Some backup storage locations are invalid

### ISSUE THREAD

**Issue by Loken3** [heptio/velero#1683]
**What steps did you take and what happened:**
[A clear and concise description of what the bug is, and what commands you ran.)
Velero install makes an error during the deployment ("some backup storage locations are invalid") with external Minio storage (I referenced his url in Velero install)

**What did you expect to happen:**
Whate I expected is that Velero deployment is ready without error

**The output of the following commands will help us better understand what's going on**:
(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)

* `kubectl logs deployment/velero -n velero`

![Capture-dcran-de-2019-07-19-15-19-16-ConvertImage](https://user-images.githubusercontent.com/18683068/61537974-b3074780-aa38-11e9-956c-47af5ec3c6c7.png)

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

My install command was : 
`velero install \
    --provider aws \
    --bucket velero \
    --secret-file ./credentials-velero \
    --use-volume-snapshots=false \
    --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://172.17.0.2:9000`

And my minio server run on Docker container in the same computer : 

![Capture-dcran-de-2019-07-19-15-12-42-ConvertImage](https://user-images.githubusercontent.com/18683068/61537681-05943400-aa38-11e9-9256-b10ff1adf5b8.png)

**Environment:**

- Velero version (use `velero version`): v1.0.0
- Kubernetes version (use `kubectl version`): v1.15.1
- Kubernetes installer & version: Minikube v1.2.0
- Cloud provider or hardware configuration: Minio
- OS (e.g. from `/etc/os-release`): Ubuntu 18.04.1 LTS

**Comment by skriss** [comment]
@Loken3 it looks like the connection to minio is not working - perhaps there's a networking issue? Can you confirm that you can manually connect to minio from another non-velero pod in the cluster, using that IP address?

**Comment by Loken3** [comment]
Thanks for helping. I don't have a network issue (all turn on one computer) but i don't tried to manually connect to minio from another non-velero pod (I just have test to ping minio server with another pod). I don't know it was possible to try to connect manually to minio to test connectivity ...

**Comment by prydonius** [comment]
@Loken3 if you're running minio in Docker and then velero in Minikube, I don't think they'll be able to connect to each other. Are you able to run minio in minikube? You can deploy the example we have in https://github.com/heptio/velero/tree/master/examples/minio

**Comment by Loken3** [comment]
Yes I already have done this configuration but I want to extract the storage system from the cluster, I hope it is possible

**Comment by Loken3** [comment]
Hi, you were right, I don't have connectivity between my cluster and my external minio container. I tried to run external minio without container and it works. I also tried to connect me to minio with an other pods in my cluster and I have too a connection refused. I am disappointed, my goal is to make a cluster with a backup/restore system and an external storage (not provider) in CoreOS where all run in container
